### DISCUSSION TITLE
getFieldState(name).invalid return deprecated message

### DISCUSSION THREAD

**Discussion by berradakamal** [react-hook-form/react-hook-form#8906] [Q&A] [answered]
Hello, 

When I use GetFieldState(name).invalid typescript returns "Depecreated" but not isDirty - isTouched - errors, the error comes from the file `react-hook-form/dist/types/form.d.ts` on line 246

Will only invalid be removed or getFieldState globally ? 

Thanks by advance for you response

**Comment by leapful** [comment, marked as answer]
@berradakamal : Only "invalid" is deprecated. 

Please check the following documentation for details:

https://github.com/react-hook-form/react-hook-form/pull/8202/files
https://github.com/react-hook-form/react-hook-form/issues/8195
